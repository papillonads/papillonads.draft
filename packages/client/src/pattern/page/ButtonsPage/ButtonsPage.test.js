import React from 'react'
import buttonsPageDefault from './ButtonsPage.render'

jest.mock('react-apollo', () => {
  return {
    Query: () => <div />,
  }
})

describe('<ButtonsPage />', () => {
  describe('Snaphot', () => {
    test('must match buttonsPageDefault', () => {
      expect(global.renderToJSON(buttonsPageDefault)).toMatchSnapshot()
    })
  })
})
