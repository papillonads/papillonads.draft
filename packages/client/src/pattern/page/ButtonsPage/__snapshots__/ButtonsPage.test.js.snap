// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonsPage /> Snaphot must match buttonsPageDefault 1`] = `<div />`;
